docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -101,7 +102,7 @@ If you are overriding `Auth.signUp`, update the override function call to includ
 
 ```
 
-#### 6. Legacy i18n translation keys removed
+#### 6. `@aws-amplify/ui-react@v4` removes legacy i18n translation keys removed
```

Comment:
`v4` or `4.x`?

Reply:
Thank you for the catch, that's a copy and paste error.
```suggestion
#### 6. `@aws-amplify/ui-react@4.x` removes legacy i18n translation keys removed
```